# Generated for NaoQi method ALAudioPlayer.playFileFromPositionString
# Plays a wav or mp3 file from a given position in the file, with specific volume and audio balance. Asynchronous.

# Name of the sound file
string fileName
# Position in second where the playing has to begin
float32 position
# volume of the sound file (must be between 0.0 and 1.0)
float32 volume
# audio balance of the sound file (-1.0 : left / 1.0 : right)
float32 pan
---
# A string that summarises the usage of the method.
int32 value


